Formula/png++.rb

```diff
@@ -14,7 +14,61 @@ class Pngxx < Formula
 
   depends_on "libpng"
 
+  # Fix strerror_r usage for OS X clang
+  # patch from <https://savannah.nongnu.org/bugs/?46312>
+  patch :DATA
```

Comment:
Would be good to submit an actual patch to them and a more detailed bug report. Thanks!